subsys/bluetooth/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ zephyr_sources_ifdef(CONFIG_BT_SCAN scan.c)
 zephyr_sources_ifdef(CONFIG_BT_CONN_CTX conn_ctx.c)
 
 add_subdirectory_ifdef(CONFIG_BT_LL_NRFXLIB controller)
+add_subdirectory_ifdef(CONFIG_BT_MESH mesh)
```

Comment:
Should this be CONFIG_BT_MESH_NORDIC_MODELS?

Reply:
Good question, I suppose so. Long term, I think this is the right define to check, but since this folder only contains models for now, it could.